official/vision/beta/projects/yolo/configs/decoders.py

```diff
@@ -0,0 +1,28 @@
+"""Backbones configurations."""
```

Comment:
this would deviate from the main vison beta structure, I feel merging them into one file will make things harder to find especially if newer yolo models like YOLOF implement a new decoder.